src/hera/workflows/_mixins.py

```diff
@@ -495,6 +495,12 @@ class TemplateMixin(SubNodeMixin, HookMixin, MetricsMixin):
     timeout: Optional[str] = None
     tolerations: Optional[List[Toleration]] = None
 
+    @validator("active_deadline_seconds")
+    def _convert_active_deadline_seconds(cls, v) -> Optional[IntOrString]:
+        if v is None or isinstance(v, IntOrString):
+            return v
+        return IntOrString(__root__=v)
```

Comment:
@flaviuvadan this is where we convert the input, regardless of whether it is int or string.